draft-ietf-intarea-provisioning-domains.md

```diff
@@ -909,42 +940,86 @@ of the additional information has passed).
-between the host and the default routers with HTTP over TLS
-communications which servers are authenticated as rightful owners of the
-FQDN received within the trusted PvD ID RA option.
+security model, but simply validates that the owner of the PvD FQDN
+authorizes its use with the prefix advertised by the router. In
+combination with implicit trust in the local router (if present), this
+gives the host some level of assurance that the PvD is authorized for
+use in this environment. However, when the local router cannot be
+trusted, no such guarantee is available.
 
 It must be noted that {{misconfig}} of this document
 only provides reasonable assurance against misconfiguration but does not
-prevent an hostile network access provider to advertise wrong
+prevent a hostile network access provider from advertising incorrect
 information that could lead applications or hosts to select a hostile PvD.
-
-Users cannot be assumed to be able to meaningfully differentiate between
-"safe" and "unsafe" networks. This is a known attack surface that is present
-whether or not PvDs are in use, and hence cannot be addressed by this document.
-However, a host that correctly implements the multiple PvD architecture ({{?RFC7556}})
-using the mechanism described in this document will be less susceptible to such
-attacks than a host that does not by being able to check for the various
-misconfigurations described in this document.
+However, a host that correctly implements the multiple PvD architecture ({{!RFC7556}})
+using the mechanism described in this document will be less susceptible to
+some attacks than a host that does not by being able to check for the various
+misconfigurations or inconsistencies described in this document.
+
+Since expiration times provided in PvD Additional Information use absolute time,
+these values can be skewed for hosts without an accurate time base, or
+due to clock skew. Such time values MUST NOT be used for security-sensitive
+functionality or decisions.
+
+An attacker generating RAs on a local network can use the H-flag and the PvD ID
+to cause hosts on the network to make requests for PvD Additional Information
+from servers. This can become a denial-of-service attack, in which an attacker
+can amplify its attack by triggering TLS connections to arbitrary servers in response
+to sending UDP packets containing RA messages. To mitigate this attack, hosts
+MUST:
+
+- limit the rate at which they fetch a particular PvD's Additional Information;
+- limit the rate at which they fetch any PvD Additional Information on a given local
+network;
+- stop making requests for a PvD ID that does not respond with valid JSON;
+- stop making requests for all PvD IDs once a certain number of failures is reached
+on a particular network.
+
+Details are provided in {{retr}}. This attack can be targeted at generic web servers,
+in which case the host behavior of stopping requesting for any server that doesn't
```

Comment:
There is also another (in our document) statement that the PvD server must NOT accept HTTPS connections coming from outside 'its' prefixes

Reply:
Indeed! I reference that later in this paragraph as {{serverop}}